app/assets/javascripts/beak/widgets/ractives/code-editor.coffee

```diff
@@ -6,6 +6,7 @@ window.RactiveModelCodeComponent = Ractive.extend({
   , lastCompiledCode:  undefined # String
   , lastCompileFailed:     false # Boolean
   , procedureNames:           {}
+  , autoCompleteStatus:  Boolean
```

Comment:
This can just be `false` to start, the `Boolean` object is the constructor for object booleans in Javascript.

Reply:
Okay.

Reply:
Will take few mins to push and squash